src/test/java/TestLIC8.java

```diff
@@ -0,0 +1,210 @@
+                0,
+                0,
+                0,
+                0,
+                0
+        );
+
+        Point[] pts = {
+                new Point(0,0)
+        };
+
+        TestUtils.checkErrorMessage(
+                () -> Decide.LIC8(pts, params),
+                "A_PTS must be greater or equal to 1"
+        );
+    }
+
+  @Test(expected = IllegalArgumentException.class)
+    public void TestIllegalB() {
+        Parameters params = new Parameters(
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                1, //Only interesting parameters for LIC8
+                0, //Only interesting parameters for LIC8
+                0,
+                0,
+                0,
+                0,
+                0
+        );
+
+        Point[] pts = {
+                new Point(0,0)
+        };
+
+        TestUtils.checkErrorMessage(
+                () -> Decide.LIC8(pts, params),
+                "B_PTS must be greater or equal to 1"
+        );
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void TestIllegalAB() {
+        Parameters params = new Parameters(
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0, //Only interesting parameters for LIC8
+                0, //Only interesting parameters for LIC8
+                0,
+                0,
+                0,
+                0,
+                0
+        );
+
+        Point[] pts = {
+                new Point(0,0)
+        };
+
+        TestUtils.checkErrorMessage(
+                () -> Decide.LIC8(pts, params),
+                "A_PTS and B_PTS must be greater or equal to 1"
+        );
+    }
+
+    @Test
+    public void TestFewerThanFivePoints() {
+        Parameters params = new Parameters(
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                0,
+                1, //Only interesting parameters for LIC8
+                1, //Only interesting parameters for LIC8
+                0,
+                0,
+                0,
+                0,
+                0
+        );
+
+        Point[] pts = {
+                new Point(0,0),
+                new Point(1,1),
+                new Point(2,2),
+                new Point(3,3),
+        };
+
+        Assert.assertFalse(Decide.LIC8(pts, params));
+    }
+
+    @Test
+    public void TestPositive() {
+        Parameters params = new Parameters(
+                0,
+                0,
+                2,
```

Comment:
This is changed, but is not mentioned as interesting for the test. Should perhaps be commented since you mention explicit which parameters that affects the test.